WhlgPortalWebsite.Data/DataAccessProvider.cs

```diff
@@ -207,4 +207,13 @@ public async Task<Consortium> GetConsortiumByConsortiumCodeAsync(string consorti
         return await context.Consortia
             .SingleAsync(c => c.ConsortiumCode == consortiumCode);
     }
+
+    public async Task DeleteUserAsync(int userId)
+    {
+        var user = await context.Users
+            .SingleAsync(u => u.Id == userId);
+
+        context.Users.Remove(user);
```

Comment:
to check also, what happens when we delete a user? will it be handled appropriately if they sign back in?